faiss/gpu/impl/L2Select.cu

```diff
@@ -256,7 +256,15 @@ void runL2SelectMin(
         } else if (k <= 1024) {
             RUN_L2_SELECT(128, 1024, 8);
 
-#if GPU_MAX_SELECTION_K >= 2048
+#if defined USE_NVIDIA_CUVS
+
+#if GPU_MAX_SELECTION_K >= 16384
```

Comment:
this code is Faiss classic only (it is a fused k-select + L2 distance computation)